azure-pipelines.yml

```diff
@@ -1,6 +1,11 @@
+pr:
+  branches:
+    include: [ 'master', 'release' ]
+    exclude: [ 'refs/tags/*' ]
```

Comment:
You can't PR to refs/tags. Could we use the shorthand `pr: [ '*' ]` so that builds run no matter what branch you pick as the PR target branch?

Reply:
Sure. I overlooked that shorthand, so thanks for the link 👍